integr8ly/installation
# generate gitea token for gitea cr and set webapp gitea token env var

The following changes includes:
- Generating a token to be used for the Gitea `INTERNAL_TOKEN`
- Set value for the `giteaInternalToken` parameter in the Gitea custom resource
- Set webapp Gitea token env var

Verification
1. Run the Gitea/install all playbook.
2. Verify that a `GITEA_TOKEN` env var is set in the web app deployment config.
3. Verify that it sets the gitea token in the `gitea-config` configmap and webapp env var only when a custom resource is created.
